gamestonk_terminal/portfolio/port_controller.py

```diff
@@ -45,59 +49,41 @@ def __init__(self):
         self.broker_list = set()
         self.merged_holdings = None
 
-    @staticmethod
-    def print_help(broker_list):
-
+    @classmethod
+    def print_help(cls, broker_list):
         """Print help"""
-        print("\nBrokers Supported: rh   - Robinhood")
-        print("                   alp  - Alpaca")
-        print("                   ally - Ally Invest")
-        print("                   dg - Degiro")
-        print("\nPortfolio:")
-        print("   help          show this menu again")
-        print(
-            "   q             quit this menu, and shows back to main menu, logs out of brokers"
-        )
-        print("   quit          quit to abandon program, logs out of brokers")
 
         print(
-            f"\nCurrent Broker: {('None', ', '.join(broker_list))[bool(broker_list)]}"
+            "Brokers Supported:\n"
+            "   ally - Ally Invest\n"
+            "   alp  - Alpaca\n"
+            "   rh   - Robinhood\n"
+            "\nPortfolio:\n"
+            "   help          show this menu again\n"
+            "   q             quit this menu, and shows back to main menu, logs out of brokers\n"
+            "   quit          quit to abandon program, logs out of brokers\n"
         )
-        print("")
-        print("   login         login to your brokers")
-
-        print("\nRobinhood:")
-        print("   rhhold        view rh holdings")
-        print("   rhhist        plot rh portfolio history")
-        print("Alpaca:")
-        print("   alphold       view alp holdings")
-        print("   alphist       view alp portfolio history")
-        print("Ally:")
-        print("   allyhold      view ally holdings")
-        print("Degiro:")
-        print("   dghold        view dg holdings")
-        print("\nMerge:")
-        print("   hold          view net holdings across all logins")
-        print("")
-
-    def print_portfolio_menu(self):
+        cls.print_portfolio_menu(broker_list)
 
+    @staticmethod
+    def print_portfolio_menu(broker_list):
         print(
-            f"\nCurrent Brokers : {('None', ', '.join(self.broker_list))[bool(self.broker_list)]}"
+            f"\nCurrent Broker: {('None', ', '.join(broker_list))[bool(broker_list)]}\n\n"
+            "Ally:\n"
+            "   allyhold      view ally holdings\n"
+            "Alpaca:\n"
+            "   alphold       view alp holdings\n"
+            "   alphist       view alp portfolio history\n"
+            "Degiro:\n"
+            "   degiro        view degiro sub-menu\n"
```

Comment:
Since degiro is a menu, it should contain a  ">" to indicate that. See how fa menu shows av or fmp submenu

Reply:
Changed